### PR TITLE
chore: remove `Dropdown` and `LanguageSelector` closes #5704

### LINKED ISSUE #5704: Delete Dropdown and LanguageSelector Components
The Dropdown and LanguageSelector components are being ported to the new website design. But we will remove themfor now so as to not complicate our styles.

Remove all references to them within the current codebase. This includes:

- [ ] all files within [components/Common/Dropdown ](https://github.com/nodejs/nodejs.org/tree/main/components/Common/Dropdown )
   - [ ] if this is the last work within a higher directory, delete that too
   - [ ] perform a global search for the component. If found elsewhere, remove its usage
- [ ] all files within [components/Common/LanguageSelector](https://github.com/nodejs/nodejs.org/tree/main/components/Common/LanguageSelector)
   - [ ] if this is the last work within a higher directory, delete that too
   - [ ] perform a global search for the component. If found elsewhere, remove its usage
- [ ] all references to the component within [internationalization files](https://github.com/nodejs/nodejs.org/tree/main/i18n/locales).
   - [ ] look for `components.common.languageSelector.*` keys within `i18n/locales/{locale}.json` files
   - [ ] ensure these files remain valid JSON by being mindful of commas when deleting lines

As with all contributions, run local commands to ensure the deletion was cleanly executed:
- [ ] `npx turbo lint`
- [ ] `npx turbo format`
- [ ] `npx turbo test`
- [ ] `npx turbo build`
- [ ] `npx turbo serve`

### PR DESCRIPTION
**Pull request by moazkhan05**
## Related Issues
closes #5704  

### Check List

- [x] I have read the [Contributing Guidelines](https://github.com/nodejs/nodejs.org/blob/main/CONTRIBUTING.md) and made commit messages that follow the guideline.
- [x] I have run `npx turbo lint` to ensure the code follows the style guide. And run `npx turbo lint:fix` to fix the style errors if necessary.
- [x] I have run `npx turbo format` to ensure the code follows the style guide.
- [x] I have run `npx turbo test` to check if all tests are passing.
- [x] I've covered new added functionality with unit tests if necessary.

### CODE REVIEW

**Comment by ovflowd** [comment]
Hey there 👋 could you solve the merge conflicts?

**Comment by moazkhan05** [comment]
@ovflowd i'll resolve the conflicts

**Comment by moazkhan05** [comment]
I have resolved the merge conflicts @ovflowd

File: components/Sections/NewHeader/index.tsx
Comment (ovflowd) [inline components/Sections/NewHeader/index.tsx:1]: This PR accidentally adds this file back. Could you remove this fie?

File: components/Common/LanguageSelector/index.tsx [hunk 1]
```diff
@@ -1,68 +0,0 @@
...
-        onClick: () => {
-          // TODO: "locale changing logic yet to be implemented"
-        },
-        active: currentLocale.code === locale.code,
-      })),
-    [availableLocales, currentLocale]
-  );
-
-  const ariaLabelText = intl.formatMessage({
-    id: 'components.common.languageSelector.button.title',
```
Comment (ovflowd) [inline components/Common/LanguageSelector/index.tsx:41]: Don't forget to remove this translation key from the localisation files. Please read the instructions given here https://github.com/nodejs/nodejs.org/issues/5704 carefully :)

**Comment by ovflowd** [comment]
I took over the PR. As these were the final Components blocking the new Website Redesign.

**Review by ovflowd — APPROVED** [review]

**Pull request merged**